RecoBTag/DeepFlavour/src/sorting_modules.cc

```diff
@@ -0,0 +1,24 @@
+
+#include "RecoBTag/DeepFlavour/interface/sorting_modules.h"
+#include <iostream>
+
+namespace btagbtvdeep{
+
+std::vector<std::size_t> invertSortingVector(const std::vector<SortingClass<std::size_t> > & in){
+    std::size_t max=0;
+    for(const auto& s:in){
+        if(s.get()>max)max=s.get();
+    }
+
+    if(max>1e3){
+        throw std::runtime_error("sorting vector size more than 1k ");
```

Comment:
cms::Exception is perhaps more appropriate

Reply:
will change

Reply:
using cms::Exception from https://github.com/cms-sw/cmssw/pull/19893/commits/d4a51ecf6c8f631b5dd9b3c6736f4e7ec2c23631